src/index.html

```diff
@@ -2,16 +2,182 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form onsubmit="onSubmit()">
+      <fieldset class="wrapper">
+        <legend>Personal information</legend>
+        <label class="form-field" for="input-surname">
+          Surname:
+          <input
+            type="text"
+            class="field"
+            name="Surname"
+            id="input-surname"
+            autocomplete="off">
+        </label>
+        <label class="form-field" for="input-name">
+          Name:
+          <input
+            type="text"
+            class="field"
+            name="Name"
+            id="input-name"
+            autocomplete="off"
+            required>
```

Comment:
```suggestion
          <input
            type="text"
            class="field"
            name="Name"
            id="input-name"
            autocomplete="off"
            required
          >
```
Выравнивай закрывающую скобку тега с открывающей